### ISSUE TITLE
Do not report MA0003 for single argument methods

### ISSUE THREAD

**Issue by drieseng** [meziantou/Meziantou.Analyzer#354]
MA0003 is currently also reported for single argument methods.
For example:
```cs
var x = TimeSpan.FromSeconds(5);
```

Would it possible to introduce an option to only report this diagnostic for methods that have multiple arguments (unless the method was decorated with **RequireNamedArgumentAttribute** of course)?

**Comment by meziantou** [comment]
I've added a new configuration to set the minimum number of parameters.

https://github.com/meziantou/Meziantou.Analyzer/blob/main/docs/Rules/MA0003.md#configuration-editorconfig

````
MA0003.minimum_method_arguments = 1 
````
